sos/archive.py

```diff
@@ -342,12 +342,14 @@ def add_file(self, src, dest=None):
                 # path case
                 try:
                     shutil.copy(src, dest)
-                except IOError as e:
+                except (IOError, OSError) as e:
                     # Filter out IO errors on virtual file systems.
                     if src.startswith("/sys/") or src.startswith("/proc/"):
                         pass
                     else:
                         self.log_info("caught '%s' copying '%s'" % (e, src))
+                    if e.errno == 2:
```

Comment:
Please use a constant from the `errno` module for this; it's already imported in `archive` so you can just replace the 2 with `errno.ENOENT`.

Reply:
errno removed as all OSErrors should be logged